filebeat/tests/integration/systemlogs_linux_test.go

```diff
@@ -50,10 +43,13 @@ func TestSystemLogsCanUseJournaldInput(t *testing.T) {
 
 	// As the name says, we want this folder to exist bu t be empty
 	globWithoutFiles := filepath.Join(filebeat.TempDir(), "this-folder-does-not-exist")
-	yamlCfg := fmt.Sprintf(systemModuleCfg, globWithoutFiles, workDir)
```

Comment:
I noticed that on my Draft PR, when I was testing the VM image, this function `TestSystemLogsCanUseJournaldInput` failed and, with a retry, worked. 

So, this is going to be marked as flaky.

Reply:
That's the big puzzle I'm trying to solve, why it only fails on CI.